.gitignore

```diff
@@ -361,3 +361,5 @@ MigrationBackup/
 
 # Config Json file
 **/config.json
+
+.idea/
```

Comment:
Can you add a comment here and explain the purpose of adding it if it's not added mistakenly?